roles/ovirt-image-template/tasks/qcow2_image.yml

```diff
@@ -112,6 +102,27 @@
     tags:
       - ovirt-template-image
 
+  - block:
+    - name: Resize disk
```

Comment:
@machacekondra Ansible-playbook is generating an error here:
```
$ ansible-playbook -e@test/ocp-vars.yaml playbooks/ovirt-vm-infra.yaml 
ERROR! no action detected in task. This often indicates a misspelled module name, or incorrect module path.

The error appears to have been in '/home/cwilkers/ovirt-ansible/roles/ovirt-image-template/tasks/qcow2_image.yml': line 106, column 7, but may
be elsewhere in the file depending on the exact syntax problem.

The offending line appears to be:

  - block:
    - name: Resize disk
      ^ here


```

Reply:
You have to use Ansible 2.4, because `ovirt_disks` was renamed to `ovirt_disk`. `ovirt_disks` can still be used but it generated warning in 2.4 and will be removed in 2.8

Reply:
That's a problem, as I'm trying to use GA RHEL code to perform the reference architecture... I have 2.3.2.0.

We can go ahead with the ovirt_disks for backwards compatibility, but if this module now requires Ansible >= 2.4, that may cause some headaches down stream.

Reply:
This will go for RHV/oVirt 4.2, where Ansible 2.4 will be for sure. If I will backport also to 4.1.z, I will need to change the name.